fulp_modules/features/species/beefmen/mob/beefman.dm

```diff
@@ -247,29 +263,51 @@
 		user.put_in_hands(dropped_meat)
 	return TRUE
 
-/datum/species/beefman/spec_attacked_by(obj/item/meat_slab, mob/living/user, obj/item/bodypart/affecting, mob/living/carbon/human/beefboy)
-	if(affecting || !istype(meat_slab, /obj/item/food/meat/slab))
```

Comment:
keep affecting

Reply:
> keep affecting

i tried keeping it but then it doesnt work when you target the mouth

Reply:
> keep affecting

so I just put it down in the bottom when the limbs are being targetted